2022-09-04 14:43:23,160 INFO    MainThread:34640 [wandb_setup.py:_flush():75] Loading settings from /home/ustc/.config/wandb/settings
2022-09-04 14:43:23,160 INFO    MainThread:34640 [wandb_setup.py:_flush():75] Loading settings from /home/ustc/ls/tcd_code/wandb/settings
2022-09-04 14:43:23,160 INFO    MainThread:34640 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-09-04 14:43:23,160 INFO    MainThread:34640 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program': 'main.py'}
2022-09-04 14:43:23,161 INFO    MainThread:34640 [wandb_init.py:_log_setup():405] Logging user logs to /home/ustc/ls/tcd_code/wandb/run-20220904_144323-8tq948cw/logs/debug.log
2022-09-04 14:43:23,161 INFO    MainThread:34640 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ustc/ls/tcd_code/wandb/run-20220904_144323-8tq948cw/logs/debug-internal.log
2022-09-04 14:43:23,161 INFO    MainThread:34640 [wandb_init.py:init():439] calling init triggers
2022-09-04 14:43:23,161 INFO    MainThread:34640 [wandb_init.py:init():443] wandb.init called with sweep_config: {}
config: {}
2022-09-04 14:43:23,161 INFO    MainThread:34640 [wandb_init.py:init():492] starting backend
2022-09-04 14:43:23,161 INFO    MainThread:34640 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-09-04 14:43:23,173 INFO    MainThread:34640 [backend.py:ensure_launched():219] starting backend process...
2022-09-04 14:43:23,183 INFO    MainThread:34640 [backend.py:ensure_launched():225] started backend process with pid: 34741
2022-09-04 14:43:23,185 INFO    MainThread:34640 [wandb_init.py:init():501] backend started and connected
2022-09-04 14:43:23,191 INFO    MainThread:34640 [wandb_init.py:init():565] updated telemetry
2022-09-04 14:43:23,192 INFO    MainThread:34640 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-09-04 14:43:23,513 WARNING MainThread:34640 [wandb_init.py:init():975] interrupted
Traceback (most recent call last):
  File "/home/ustc/anaconda3/envs/lhc/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 954, in init
    run = wi.init()
  File "/home/ustc/anaconda3/envs/lhc/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 597, in init
    run_result = backend.interface.communicate_run(run, timeout=30)
  File "/home/ustc/anaconda3/envs/lhc/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 210, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/home/ustc/anaconda3/envs/lhc/lib/python3.7/site-packages/wandb/sdk/interface/interface_shared.py", line 320, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/home/ustc/anaconda3/envs/lhc/lib/python3.7/site-packages/wandb/sdk/interface/interface_shared.py", line 222, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/ustc/anaconda3/envs/lhc/lib/python3.7/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/ustc/anaconda3/envs/lhc/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/home/ustc/anaconda3/envs/lhc/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
